src/main/java/core/basesyntax/machines/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.machines;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator is work");
```

Comment:
There is a grammatical error in the output string. Consider changing "Excavator is work" to "Excavator is working" for consistency and correctness.